application/models.py

```diff
@@ -370,31 +380,104 @@ class LandingPages(Page):
             heading="SELITE",
             help_text='Selite sijoittuu otsikon yläpuolelle. Voit jättää tämän kohdan myös tyhjäksi.',
         ),
+    ]
+
```

Comment:
Could you please add `edit_handler` to `BannerPages` as well so that all pages in CMS would have consistent interface tabs?

Reply:
Added, thanks for pointing it out 👌